apps/userSettings/views.py

```diff
@@ -0,0 +1,17 @@
+from django.contrib.auth.models import User
+from django.shortcuts import render
+
+# Create your views here.
+from apps.userSettings.forms import userSettingsForm
+
+
+def userSettings(request):
```

Comment:
Macht es nicht sinn hier eher die Build in FormView Klasse zu nutzen?

https://docs.djangoproject.com/en/2.0/ref/class-based-views/generic-editing/#formview

Oder eventuell noch einfacher, die UpdateView
https://docs.djangoproject.com/en/2.0/ref/class-based-views/generic-editing/#updateview